webapp/backend/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
```

Comment:
I am not an expert in java, therefor it's just a question, should we include `maven-wrapper.jar` in git?
Found on [stack](https://stackoverflow.com/questions/51792830/what-files-in-a-maven-project-should-be-committed-to-git) and [default gitignore by toptal for maven](https://www.toptal.com/developers/gitignore/api/maven) that it is usually placed in `.gitignore`.

Reply:
Question to @MateuszKepczynski

Reply:
Its a stuff genereted from java code, tehres no need to keep this on github